src/main/scala/it/pagopa/interop/authorizationserver/api/impl/AuthApiServiceImpl.scala

```diff
@@ -116,7 +117,7 @@ final case class AuthApiServiceImpl(
     contexts: Seq[(String, String)]
   ): Future[KeyWithClient] =
     authorizationManagementService
-      .getKeyWithClient(clientId, kid)(contexts.filter(_._1 == CORRELATION_ID_HEADER))
+      .getKeyWithClient(clientId, kid)(contexts.filter(c => List(CORRELATION_ID_HEADER, IP_ADDRESS).contains(c._1)))
```

Comment:
added because `getKeyWithClient` tries to retrieve the IP with `fastGetOpt(contexts)(IP_ADDRESS)`, that would always return a `None`